runtime/filetype.lua

```diff
@@ -35,16 +35,16 @@ end
 -- Set a marker so that the ftdetect scripts are not sourced a second time by filetype.vim
 vim.g.did_load_ftdetect = 1
 
--- If filetype.vim is disabled, set up the autocmd to use scripts.vim
+-- If filetype.vim is disabled, set up the autocmd to use scripts.lua
 if vim.g.did_load_filetypes then
```

Comment:
Shouldn't this be `if vim.g.did_load_filetypes ~= 0`?

Reply:
no, since `g:did_load_filetypes = 0` is used to disable `filetype.vim` but still run `filetype.lua`.

(This will be changed by the _next_ PR that switches to `filetype.lua` by default.)